common/trainers/classification_trainer.py

```diff
@@ -44,18 +44,17 @@ def train_epoch(self, epoch):
                 else:
                     scores = self.model(batch.text[0], lengths=batch.text[1])
 
-            if 'single_label' in self.config and self.config['single_label']:
-                for tensor1, tensor2 in zip(torch.argmax(scores, dim=1), torch.argmax(batch.label.data, dim=1)):
-                    if np.array_equal(tensor1, tensor2):
-                        n_correct += 1
-                loss = F.cross_entropy(scores, torch.argmax(batch.label.data, dim=1))
-            else:
+            if 'is_multilabel' in self.config and self.config['is_multilabel']:
```

Comment:
`if self.config.get('is_multilabel')`?

Reply:
this is same as the hasattr() test that we talked about. Is there an elegant workaround for this?

Reply:
No, this is different from the one we talked about. We talked about TAR attribute. 

But this also needs to be fixed with the get() method with a default return value in case the key isn't present.